cms/src/test/java/uk/nhs/digital/apispecs/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest.java

```diff
@@ -72,6 +72,33 @@ public void getHtmlForSpec_throwsException_onCodeGenFailure() {
         // expectations set in 'given' are satisfied
     }
 
+    @Test
+    public void hideRequestHeaderWhenSectionIsEmpty() {
+
+        // given
+        final String specificationJson = apigeeApiSpecificationJsonWithEmptyReqSection();
+
+        // when
+        final String actualSpecHtml = swaggerCodeGenApiSpecHtmlProvider.htmlFrom(specificationJson);
+
+        // then
+        final String expectedSpecHtml = customisedSwaggerCodeGenGeneratedSpecificationHtmlForEmptyReqSection();
+
+        assertThat(
+            "Specification HTML has been generated using customised Swagger CodeGen v3",
+            actualSpecHtml,
+            is(expectedSpecHtml)
+        );
+    }
+
```

Comment:
Given that we have the above test case, we need to demonstrate the same for Response and Schema sections.

Given that the Schema is rendered through the `body_params` template, the `BodyRenderTest` class is probably a better fit for testing Schema section.

Reply:
Fixed